src/main/flight/gps_rescue.c

```diff
@@ -72,6 +72,7 @@ typedef enum {
     RESCUE_HEALTHY,
     RESCUE_FLYAWAY,
     RESCUE_CRASH_DETECTED,
+    RESCUE_STALLED,
```

Comment:
Looking through the code, `RESCUE_CRASH_DETECTED` should probably be renamed into `RESCUE_CRASH_FLIP_DETECTED`, as that is what it is used for after this change.